src/styles/Theme.tsx

```diff
@@ -65,16 +65,16 @@ const theme: DefaultTheme = {
         height: '1',
       },
       previewTitle: {
-        min: '18px',
-        size: '26px',
-        max: '1.2rem',
+        min: '16px',
+        size: '24px',
```

Comment:
if the text does not need to grow/shrink with window size, consider hardcoding the value instead of using clamp; else size should be a relative value not absolute!

Reply:
Resolved in 3801085